spec/helpers/gitlab_markdown_helper_spec.rb

```diff
@@ -564,91 +578,94 @@ def url_helper(image_name)
+      expect(markdown("\nDon't use `$#{snippet.id}` here.\n")).to eq(
         "<p>Don't use <code>$#{snippet.id}</code> here.</p>\n"
+      )
     end
 
     it "should leave ref-like autolinks untouched" do
-      markdown("look at http://example.tld/#!#{merge_request.iid}").should == "<p>look at <a href=\"http://example.tld/#!#{merge_request.iid}\">http://example.tld/#!#{merge_request.iid}</a></p>\n"
+      expect(markdown("look at http://example.tld/#!#{merge_request.iid}")).to eq("<p>look at <a href=\"http://example.tld/#!#{merge_request.iid}\">http://example.tld/#!#{merge_request.iid}</a></p>\n")
     end
 
     it "should leave ref-like href of 'manual' links untouched" do
-      markdown("why not [inspect !#{merge_request.iid}](http://example.tld/#!#{merge_request.iid})").should == "<p>why not <a href=\"http://example.tld/#!#{merge_request.iid}\">inspect </a><a class=\"gfm gfm-merge_request \" href=\"#{project_merge_request_url(project, merge_request)}\" title=\"Merge Request: #{merge_request.title}\">!#{merge_request.iid}</a><a href=\"http://example.tld/#!#{merge_request.iid}\"></a></p>\n"
+      expect(markdown("why not [inspect !#{merge_request.iid}](http://example.tld/#!#{merge_request.iid})")).to eq("<p>why not <a href=\"http://example.tld/#!#{merge_request.iid}\">inspect </a><a class=\"gfm gfm-merge_request \" href=\"#{project_merge_request_url(project, merge_request)}\" title=\"Merge Request: #{merge_request.title}\">!#{merge_request.iid}</a><a href=\"http://example.tld/#!#{merge_request.iid}\"></a></p>\n")
     end
 
     it "should leave ref-like src of images untouched" do
-      markdown("screen shot: ![some image](http://example.tld/#!#{merge_request.iid})").should == "<p>screen shot: <img src=\"http://example.tld/#!#{merge_request.iid}\" alt=\"some image\"></p>\n"
+      expect(markdown("screen shot: ![some image](http://example.tld/#!#{merge_request.iid})")).to eq("<p>screen shot: <img src=\"http://example.tld/#!#{merge_request.iid}\" alt=\"some image\"></p>\n")
     end
 
     it "should generate absolute urls for refs" do
-      markdown("##{issue.iid}").should include(project_issue_url(project, issue))
+      expect(markdown("##{issue.iid}")).to include(project_issue_url(project, issue))
     end
 
     it "should generate absolute urls for emoji" do
-      markdown(':smile:').should(
+      expect(markdown(':smile:')).to(
         include(%(src="#{Gitlab.config.gitlab.url}/assets/emoji/smile.png))
       )
     end
 
     it "should generate absolute urls for emoji if relative url is present" do
-      Gitlab.config.gitlab.stub(:url).and_return('http://localhost/gitlab/root')
-      markdown(":smile:").should include("src=\"http://localhost/gitlab/root/assets/emoji/smile.png")
+      allow(Gitlab.config.gitlab).to receive(:url).and_return('http://localhost/gitlab/root')
+      expect(markdown(":smile:")).to include("src=\"http://localhost/gitlab/root/assets/emoji/smile.png")
```

Comment:
Line is too long. [105/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.